modules/nw-ptp-introduction.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-ptp.adoc
+
+[id="ptp-introduction_{context}"]
+= Introduction to PTP
+
+The Precision Time Protocol (PTP) is a protocol used to synchronize clocks in a network. When used in conjunction with hardware support, PTP is capable of sub-microsecond accuracy, which is far more accurate than Network Time Protocol (NTP).
+
+The linuxptp package includes the `ptp4l` and `phc2sys` programs for clock synchronization. The `ptp4l` program implements the PTP boundary clock and ordinary clock. The `ptp4l` program synchronizes the PTP hardware clock to the source clock with hardware time stamping and synchronizes the system clock to the source clock with software time stamping. The `phc2sys` program is only needed for hardware time stamping to synchronize the system clock to the PTP hardware clock on the network interface card (NIC).
+
+[id="ptp-elements_{context}"]
+== Elements of a PTP domain
+
+PTP included multiple nodes connected through the network, each representing a clock. The following type of clocks can be included in configurations:
+
+Grandmaster clock:: The grandmaster clock is the reference clock that provides standard time information to other clocks across the network.  The grandmaster clock needs to be able to ensure accurate and stable synchronisation. The grandmaster clock writes time stamps and responds to time requests from other clocks.
+
+Ordinary clock:: The ordinary clock has a single port connection that can play the role of source or destination clock, depending on its position in the network. The ordinary clock can read and write time stamps.
```

Comment:
"Ordinary clock:: Has a single port connection that can play the role of source or destination clock, depending on its position in the network. The ordinary clock can read and write time stamps."